skyportal/handlers/api/weather.py

```diff
@@ -60,11 +68,19 @@ def get(self):
                               type: string
                               description: Weather fetching error message
         """
-        user_prefs = getattr(self.current_user, 'preferences', None) or {}
+        user = (
```

Comment:
Under what circumstances can a user not access their own user prefs object?

Reply:
You mean the `query_records_accessible_by...` part? Yeah that's unnecessary and is just for consistency among handler queries